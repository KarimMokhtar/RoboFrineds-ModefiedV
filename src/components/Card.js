import React from "react";

const Card = (props) => {
  const { robot } = props;
  return (
    <div className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5">
      <img
        src={`https://robohash.org/${robot.name}?size=200x200`}
        alt={robot.name}
      />
      <div>
        <h2>{robot.name}</h2>
        <p>{robot.email}</p>
      </div>
    </div>
  );
};

export default Card;
