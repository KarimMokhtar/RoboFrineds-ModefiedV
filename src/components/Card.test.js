import React from "react";
import { shallow } from "enzyme";
import Card from "./Card";

it("expect to render card component", () => {
  const wrapper = shallow(<Card name="test" email="email@tet.com" />);
  expect(wrapper.debug()).toMatchSnapshot();
});
