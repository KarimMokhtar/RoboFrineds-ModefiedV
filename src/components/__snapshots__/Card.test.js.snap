// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
"<div className=\\"tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5\\">
  <img src=\\"https://robohash.org/test?size=200x200\\" alt=\\"test\\" />
  <div>
    <h2>
      test
    </h2>
    <p>
      email@tet.com
    </p>
  </div>
</div>"
`;
